app/src/main/res/layout/fragment_account_management_settings.xml

```diff
@@ -0,0 +1,130 @@
+
+            <TextView
+                android:id="@+id/userNameTitle"
+                style="@style/BodyMedium"
+                android:layout_width="match_parent"
+                android:layout_height="wrap_content"
+                android:layout_marginHorizontal="@dimen/marginStandardMedium"
+                android:layout_marginTop="@dimen/marginStandard"
+                android:text="@string/usernameTitle"
+                app:layout_constraintTop_toTopOf="parent" />
+
+            <TextView
+                android:id="@+id/username"
+                style="@style/Body"
+                android:layout_width="match_parent"
+                android:layout_height="wrap_content"
+                android:layout_marginHorizontal="@dimen/marginStandardMedium"
+                app:layout_constraintTop_toBottomOf="@id/userNameTitle"
+                tools:text="Test" />
+
+            <TextView
+                android:id="@+id/emailTitle"
+                style="@style/BodyMedium"
+                android:layout_width="match_parent"
+                android:layout_height="wrap_content"
+                android:layout_marginHorizontal="@dimen/marginStandardMedium"
+                android:layout_marginTop="@dimen/marginStandardMedium"
+                android:text="@string/attachMailboxInputHint"
+                app:layout_constraintTop_toBottomOf="@id/username" />
+
+            <TextView
+                android:id="@+id/email"
+                style="@style/Body"
+                android:layout_width="match_parent"
+                android:layout_height="wrap_content"
+                android:layout_marginHorizontal="@dimen/marginStandardMedium"
+                android:paddingBottom="@dimen/marginStandardMedium"
+                app:layout_constraintTop_toBottomOf="@id/emailTitle"
+                tools:text="test@noreply.com" />
+
+            <com.google.android.material.divider.MaterialDivider
+                android:id="@+id/divider"
+                style="@style/ItemDivider"
+                app:layout_constraintEnd_toEndOf="parent"
+                app:layout_constraintStart_toStartOf="parent"
+                app:layout_constraintTop_toBottomOf="@id/email" />
+
+            <com.google.android.material.button.MaterialButton
+                android:id="@+id/deleteAccountButton"
+                style="@style/TextButtonSecondary"
+                android:layout_width="wrap_content"
+                android:layout_height="wrap_content"
+                android:fontFamily="@font/suisseintl_regular"
+                android:paddingHorizontal="@dimen/mediumButtonHorizontalMargin"
+                android:text="@string/buttonAccountDelete"
+                android:textColor="@color/red_light"
+                app:icon="@drawable/ic_bin"
+                app:iconTint="@color/red_light"
+                app:layout_constraintStart_toStartOf="parent"
+                app:layout_constraintTop_toBottomOf="@id/divider" />
```

Comment:
The colors of the button are not correct, don't use primitive colors, instead use a color that adapts depending on the current light/dark theme

Reply:
Also use a button that will ripple from one side of the screen to the other